langkit/templates/pkg_rewriting_impl_body_ada.mako

```diff
@@ -841,22 +1027,30 @@ package body ${ada_lib_name}.Rewriting_Implementation is
    function Create_Regular_Node
      (Handle   : Rewriting_Handle;
       Kind     : ${root_node_kind_name};
-      Children : Node_Rewriting_Handle_Array) return Node_Rewriting_Handle
-   is
-      Result : Node_Rewriting_Handle := Allocate
-        (Kind, Handle, No_Unit_Rewriting_Handle, No_Node_Rewriting_Handle);
+      Children : Node_Rewriting_Handle_Array) return Node_Rewriting_Handle is
    begin
-      Result.Children := (Kind   => Expanded_Regular,
-                          Vector => <>);
-      Result.Children.Vector.Reserve_Capacity (Children'Length);
+      ${pre_check_rw_handle("Handle")}
+      ${pre_check_is_not_token_kind("Kind")}
       for C of Children loop
```

Comment:
Given that `C` will appear in the error message, please rename it so something more explicit, like `One_Child`.

Likewise for `A` in Create_From_Template.